libs/ui-lib/lib/common/components/ui/formik/validationSchemas.ts

```diff
@@ -507,14 +507,12 @@ const httpProxyValidationMessage = 'Provide a valid HTTP URL.';
 export const httpProxyValidationSchema = (
   values: ProxyFieldsType,
   pairValueName: 'httpProxy' | 'httpsProxy',
-) =>
-  Yup.string()
-    .test(
-      'http-proxy-no-empty-validation',
-      'At least one of the HTTP or HTTPS proxy URLs is required.',
-      (value) => Boolean(!values.enableProxy || value || values[pairValueName]),
-    )
-    .test('http-proxy-validation', httpProxyValidationMessage, (value: string) => {
+  allowEmpty?: boolean,
```

Comment:
About codestyle only: having an optional boolean param is not good for the maintenance since we can not add more params later and it is not a big deal to pass `false` here now.
In addition, it makes sense to review other usages of that and decide about the value there.

Reply:
I've changed the input params to be an object. Which still enables us to have optional params & add more later :)
All other usages of the `httpProxyValidationSchema` were reviewed.